EmailAuth/FirebaseEmailAuthUI/FUIPasswordSignInViewController.m

```diff
@@ -168,10 +168,10 @@ - (void)signInWithDefaultValue:(NSString *)email andPassword:(NSString *)passwor
             return;
         }
       }
-      [self.navigationController dismissViewControllerAnimated:YES completion:^{
-        if (self->_onDismissCallback) {
-          self->_onDismissCallback(authResult, error);
-        }
+      [self dismissNavigationControllerAnimated:true completion:^{
+          if (self->_onDismissCallback) {
+              self->_onDismissCallback(authResult, error);
+          }
```

Comment:
Nits: 
- Use two space indents
- Use `YES` in ObjC, `true` in Swift

Other than that, this change looks good.

Reply:
Thank you for corrections, fixed both points.